src/components/Modal/Modal.stories.js

```diff
@@ -24,12 +24,42 @@ export default {
         },
     },
     parameters: {
+        docs: {
+            // opt-out of inline rendering
```

Comment:
Opting out of inline rendering of modals for now so that docs can be at least visible... the demos on the docs page are currently broken due to the iframe sizes. Needs Fixing (iframe size).